style_guide.md

```diff
@@ -0,0 +1,118 @@
+# Style Guide
+
+## What is this?
+This is a set of guidelines on how to *style* code you plan on contributing to the prog-o-meter repository. You don't have to follow each, or any of them, but it may take your PR longer to get merged if someone else has to rewrite and document your code.
```

Comment:
after style, I think it should say 
any code or the code for a more grammatical sentence

Reply:
The second sentence is misleading, since I don't rewrite people's code. I just give them instructions on how to get it right, because otherwise they'll make the same mistake in the future. I suggest that we add some text from the original guide instead, so the entire section becomes:

This is a set of guidelines on how to *style* [THE/ANY] code you plan on contributing to the prog-o-meter repository. Please do not let the guidelines discourage you from contributing. If you need further explanation of anything, please ask. If you are not sure your contribution follows the guidelines, __DO NOT FRET__, please submit a pull-request regardless, and we will help you make any necessary changes, before we merge. 

No one is perfect! I am 100% sure you will find things in the current code, that do not follow the guidelines below. If you do, please either open a pull request with a fix so we can improve, or let us know by opening an issue, so someone else can fix it.